cartaya/2/style.css

```diff
@@ -0,0 +1,184 @@
+@font-face {
+    font-family: 'HelveticaNeueCE35Thin';
+    src: url('Helvetica Neue CE 35 Thin.ttf');
+}
+
+* { margin: 0px;}
+
+header {
+    background-color: #447CA2;
+    height: 102px;
+    margin-top: 57px;    
+}
+
+main {
+    overflow: auto;
+    padding-bottom: 50px;
+}
+
+#h1 {
+    font-family: HelveticaNeueCE35Thin;
+    font-size: 102px;
+    letter-spacing: 2px;
+    color: #FFFFFF;
+    margin-left: 53px;
+    padding-top: 7px;
+    position: absolute;
+}
+
+.color-borde {
+    border: 2pt;
+    border-style: solid;
+    border-color: #D0CCCC;
+}
+
+.lupa {
+    padding: 6px 5px;
+}
+
+#lupa {
+    width: 24px;
+    height: 27px;
+    float: left;
+    position: relative;
+}
+
+#people-search {
+    margin-top: 28px;
+    margin-left: 53px;
+}
+
+#search {
+    width: 213px;
+    height: 32.33px;
+    padding: 1px 1px 1px 2px;
+    float: left;
+    position: relative;
+    font-family: HelveticaNeueCE35Thin;
+    font-size: 18pt;
+    color: #DFDFDF;
+}
+#boton {
+    position: relative;
+    float: left;
+    height: 32.33px;
+    width: 78px;
+    background-color: #447CA2;
+    font-family: HelveticaNeueCE35Thin;
+    color: #FFFFFF;
+    font-size: 18px;
+    text-align: left;
+    border: 0px;
+    padding: 0px 0px 0px 4px;
+    margin: 0px 0px 0px 2px;
+    cursor: pointer;
+}
+
+#boton:hover{
+    background-color: rgb(167, 167, 167);
+    transition: background-color .3s;
+}
+
+.selector-azul{
+    -webkit-appearance: checkbox;
+}
+label {

```

Comment:
you shouldn't never ever use css selector for element name, use classes ... there reason behind this is that when you apply styles directly to an element, it means all element label in you website have applied the same styles